docs/data-sources/module.md

```diff
@@ -92,4 +92,10 @@ Read-Only:
 
 Read-Only:
 
+- `description` (String)
+- `id` (String)
+- `is_default` (Boolean)
+- `labels` (Set of String)
+- `name` (String)
 - `namespace` (String)
+- `space_id` (String)
```

Comment:
Should there be something similar for `stacks.md` and `stack.md` (in `gitlab` section)?

Reply:
you're right, I dont know how I missed it 😢